src/lib/solana/manuallyExecuteSolana.ts

```diff
@@ -114,3 +146,116 @@ export function newAnchorProvider(chainName: string) {
   })
   return { anchorProvider, keypair }
 }
+
+type ManualExecAccounts = {
+  config: PublicKey
+  referenceAddresses: PublicKey
+  sourceChain: PublicKey
+  commitReport: PublicKey
+  offramp: PublicKey
+  allowedOfframp: PublicKey
+  rmnRemote: PublicKey
+  rmnRemoteCurses: PublicKey
+  rmnRemoteConfig: PublicKey
+  authority: PublicKey
+  systemProgram: PublicKey
+  sysvarInstructions: PublicKey
+}
+
+async function bufferedTransactions(
+  destinationProvider: AnchorProvider,
+  bufferProgramAddress: string,
+  serializedReport: Buffer,
+  serializedTokenIndexes: Buffer<ArrayBuffer>,
+  originalManualExecAccounts: ManualExecAccounts,
+  originalManualExecRemainingAccounts: AccountMeta[],
+  computeBudgetIx: TransactionInstruction,
+  blockhash: string,
+  addressLookupTableAccounts: AddressLookupTableAccount[],
+): Promise<VersionedTransaction[]> {
+  // Arbitrary as long as there's consistency for all translations.
+  const bufferId = {
+    bytes: Array.from(randomBytes(32)),
+  }
+  const [bufferAddress] = PublicKey.findProgramAddressSync(
+    [
+      Buffer.from('execution_buffer'),
+      destinationProvider.wallet.publicKey.toBuffer(),
+      Buffer.from(bufferId.bytes),
+    ],
+    new PublicKey(bufferProgramAddress),
+  )
+
+  const chunkSize = 250
+
+  const bufferingProgram = new Program(
+    EXECUTION_BUFFER_IDL,
+    new PublicKey(bufferProgramAddress),
+    destinationProvider,
+  )
+
+  var transactions: VersionedTransaction[] = []
```

Comment:
Please don't use `var`. Given that the array here is just then extended in place without needed to re-assign the variable reference, you should be able to use `const` instead.